badges.md

```diff
@@ -24,6 +24,7 @@ These will be in the codebase. They're unordered.
 - **Lackadaisical:** Solve bob in five languages
 - **Team builder:** Create a team with 3 members
 - **Teammate:** Give team feedback to 5 solutions.
+- **Weekend Exerciser:** Only submit solutions on the weekends for a month
```

Comment:
[Different parts of the world have different weekends](https://en.wikipedia.org/wiki/Workweek_and_weekend#Around_the_world). Will the badge be localised or is there an implicit assumption of a "most of the world" Sat/Sun weekend?